pkg/pipelineascode/pipelineascode.go

```diff
@@ -198,9 +198,11 @@ func (p *PacRun) startPR(ctx context.Context, match matcher.Match) (*tektonv1.Pi
 
 	// Patch pipelineRun with logURL annotation, skips for GitHub App as we patch logURL while patching CheckrunID
 	if _, ok := pr.Annotations[keys.InstallationID]; !ok {
+		// save pr data for error logging in case the patch returns nil on error
+		generateName := pr.GetGenerateName()
 		pr, err = action.PatchPipelineRun(ctx, p.logger, "logURL", p.run.Clients.Tekton, pr, getLogURLMergePatch(p.run.Clients, pr))
 		if err != nil {
-			return pr, fmt.Errorf("cannot patch pipelinerun %s: %w", pr.GetGenerateName(), err)
```

Comment:
humm yeah i see why it fails now, pr.GetName() would have probably work too... but this works as well... not being able to patch the pipelinerun is the actual real problem tho....